elastic/kibana
# [Fleet] Retry Saved Object import on conflict error

## Summary

Closes #126611.

We are now encountering conflict errors when we try and install many packages at once (as they all try and create the index patterns) due to some recent changes in the SO import code. After weighing up some alternate approaches (removing the check, not creating the index patterns each time) @joshdover and I decided this was the most reliable approach.

## Manual Testing
I was able to change the code to recreate this situation, and I've added unit tests to recreate the conflict errors, but a true manual test would be: 

- on a 7.x system, install at least 10 packages
- upgrade to 8.1 and notice the packages are all force reinstalled successfully (i.e no conflict errors)

### Checklist
